### ISSUE TITLE
[Enhancement] Cache web tiles

### ISSUE THREAD

**Issue by roya0045** [MerginMaps/input#2716]
As the app can be used in area with slow or no connection, it would be nice to cache the tiles that were seen previously, either in this session or at a broader scope. Reusing the tiles could save some bandwitdh and not having to fetch all the tiles within the screen extent before refreshing the image would be useful.

Though I understand that this might depend on the type of service and that such management might be complex to implement.

**Comment by PeterPetrik** [comment]
workaround? :) https://www.linkedin.com/feed/update/urn:li:activity:7064989366737133568/

**Comment by roya0045** [comment]
> workaround? :) https://www.linkedin.com/feed/update/urn:li:activity:7064989366737133568/

Yes this is our second option, though using a single static image below the basemap might be easier and take less space as we can cover wide areas during our fieldwork. I don't want to clog up too much space on servers as redundancy.
